.github/workflows/ci_dependabot.yml

```diff
@@ -23,6 +23,5 @@ jobs:
       update_pre-commit: true
       python_version: "3.9"
       install_extras: "[dev]"
-      skip_pre-commit_hooks: pylint,pylint-tests
     secrets:
```

Comment:
I forget if ci_dependabot is something we directly control. If so it would probably be better to have a section that lets you choose the linter, e,g., linter=ruff, rather than disabling pylint and the manually setting a  ruff linter down the line. [For me this seems like a small amount of tech debt being added]

Reply:
This is a bit opaque as well.

I'll try to explain:
The callable workflows (like the one used for the `ci_dependabot.yml` file) is indeed coming from a repository we are managing and I am the main developer of: [SINTEF/ci-cd](https://SINTEF.github.io/ci-cd).

This particular called workflow works in the way that it also updates the `pre-commit` hooks (i.e., running `pre-commit autoupdate`).
Then, if any hooks were updated it runs all the hooks on all the filed (i.e., `pre-commit run --all-files`), committing any changes automagically, before opening a PR based on `permanent_dependencies_branch` (here `ci/dependency-updates`) against `default_repo_branch` (here `master`).

Due to issues with how `pylint` is implemented in the local `.pre-commit-hooks.yaml` file, running pylint hooks is problematic in the CI environment. Hence, we needed to skip running these when calling `pre-commit run --all-files`.

Now that we're removing pylint from the `.pre-commit-hooks.yaml` file, they no longer need to be skipped.

---

Essentially, the implementation of a switch from pylint (and isort) -> ruff happens in the `.pre-commit-hooks.yaml` file. Everything in the CI files are effects happening due to this change - for `ci_tests.yml` we simply skip the pre-commit test altogether - again it was run explicitly here due to the inability of running it "normally" through `pre-commit run --all-files`. If this was possible, there would be no need to run pylint explicitly.

Again here, `ruff` runs perfectly fine as part of running pre-commit, so there's no need for an explicit `ruff` execution in addition to running `pre-commit`.

Note that these last comments relate to `ci_tests.yml`, not `ci_dependabot.yml`, where this comment is otherwise attached.

Reply:
Bah misread this as a new line, rather than a removed one. Makes total sense now.